include/rand.h

```diff
@@ -0,0 +1,14 @@
+#ifndef Z_RAND_H
```

Comment:
```suggestion
#ifndef RAND_H
```
If the header filename doesn't have a z_ prefix neither should the header guard